tests/unit/lms/services/application_instance_getter_test.py

```diff
@@ -69,40 +69,38 @@ def test_provisioning_returns_False_if_consumer_key_unknown(self, ai_getter):
         assert not ai_getter.provisioning_enabled()
 
     @pytest.mark.parametrize(
-        "section_groups_feature_flag,developer_key,canvas_sections_enabled,expected_result",
+        "params",
         [
-            (True, "test_developer_key", True, True),
-            (False, "test_developer_key", True, False),
-            (True, None, True, False),
-            (True, "test_developer_key", False, False),
-            (False, None, False, False),
+            dict(),
+            dict(feature_flag=False, expected_result=False),
+            dict(developer_key=None, expected_result=False),
+            dict(canvas_sections_enabled=False, expected_result=False),
         ],
     )
     def test_canvas_sections_enabled(
-        self,
-        pyramid_request,
-        test_application_instance,
-        section_groups_feature_flag,
-        developer_key,
-        canvas_sections_enabled,
-        expected_result,
+        self, pyramid_request, test_application_instance, params,
     ):
-        def feature(flag):
-            if section_groups_feature_flag:
-                return flag == "section_groups"
-            return False
+        default_params = dict(
+            feature_flag=True,
+            developer_key="test_developer_key",
+            canvas_sections_enabled=True,
+            expected_result=True,
+        )
```

Comment:
I was going to say you could do `get(..., default)` where it's needed, but it easier to compare with the parameterization above this way.